src/main/java/net/pms/formats/v2/SubtitleType.java

```diff
@@ -47,6 +47,8 @@
 	TX3G        (11, "Timed text (TX3G)",           list(),             list("tx3g"),                                                     type.TEXT),
 	PGS         (12, "Blu-ray subtitles",           list(),             list("S_HDMV/PGS", "PGS", "144"),                                 type.PICTURE),
 	WEBVTT      (13, "WebVTT",                      list("vtt"),        list("WebVTT"),                                                   type.TEXT);
+//	EIA_608		(14, "CEA-608",						list(),				list("EIA-608"),										  		  type.TEXT),
+//	EIA_708		(14, "CEA-708",						list(),				list("EIA-708"),										  		  type.TEXT);
```

Comment:
@valib Should not be added before fixing that issue first:
https://github.com/UniversalMediaServer/UniversalMediaServer/issues/1232

As a side note, special FFmpeg command line parameter need also to be set.

Reply:
@Sami32 can you help me and provide the FFmpeg command for closed captions?

Reply:
@valib Yes, though i don't remember exactly, but i'm sure that i write it somewhere.
I cannot remember for sure, but i think it is in the FFmpeg documentation, hopefully, as many thing are still not.
That's said, right now, it's of no use, until the issue get fixed.

Reply:
@Sami32 so why we should spend time to fix #1232 if we don't know if and when it will be solved and how it will be implemented?

Reply:
@valib Yes, that what i tried to told you, no need to go further on that before solving this issue first, it t will be like put the char before the buffalo...